website/pages/docs/drivers/docker.mdx

```diff
@@ -267,11 +267,12 @@ The `docker` driver supports the following configuration in the job spec. Only
   docker daemon.
 
 - `volumes` - (Optional) A list of `host_path:container_path` strings to bind
-  host paths to container paths. Mounting host paths outside of the allocation
-  directory can be disabled on clients by setting the `docker.volumes.enabled`
-  option set to false. This will limit volumes to directories that exist inside
-  the allocation directory. We recommend using [`mounts`](#mounts) if you wish
-  to have more control over volume definitions.
+  host paths to container paths. Mounting host paths outside of the
+  [allocation working directory] can be disabled on clients by setting the
+  `docker.volumes.enabled` option set to false. This will limit volumes to
+  directories that exist inside the allocation working directory. We recommend
+  using [`mounts`](#mounts) if you wish to have more control over volume
+  definitions.
```

Comment:
Flipping around the sense from enable to disable to reflect the default setting. Trying to start pulling out enable/disable in favor of prevented (disallowed?)/allow(permitted?).  I mixed them in the text, because I don't have a clear winner.
```suggestion
  host paths to container paths. Mounting host paths outside of the [allocation
  working directory] is prevented by default and limits volumes to directories
  that exist inside the allocation working directory. You can allow mounting
  host paths outside of the [allocation working directory] on individual clients
  by setting the `docker.volumes.enabled` option to `true` in the client's
  configuration. We recommend using [`mounts`](#mounts) if you wish to have more
  control over volume definitions.
```

Reply:
Nice! 👍